godtools/Analtyics/GodToolsAnaltyics.swift

```diff
@@ -102,14 +107,17 @@ class GodToolsAnaltyics {
         
         tracker?.send(screenViewInfo)
         
-        recordScreenViewInAdobe(screenName: screenName)
+        adobeAnalyticsBackgroundQueue.async { [unowned self] () in
+            self.recordScreenViewInAdobe(screenName: screenName)
```

Comment:
Maybe use a weak self? (I don't know what is `unowned self`, will check)

Reply:
Oh I see, this is cool 👍

Reply:
Yeah a `weak` self could be nil, which would mean having to unwrap the optional. I don't see any way that self could be nil here so `unowned` should work and thus removes the optional. (I don't understand what the term "unowned" really means here, but it is the term anyways)